package.json

```diff
@@ -7,7 +7,7 @@
     "chai": "*",
     "eventemitter2": "~2.0.2",
     "google-auto-auth": "~0.2.4",
-    "grpc": "~0.15.0",
+    "grpc": "~1.0",
```

Comment:
Why does your change description not mention the grpc upgrade? This is potentially a big jump.